src/components/backend-ai-webui.ts

```diff
@@ -51,6 +51,7 @@ import {
   registerTranslateConfig,
   translate as _t,
   use as setLanguage,
+  translateUnsafeHTML as _tr,
```

Comment:
Is there a lit-translate option to keep specific HTML like`transKeepBasicHtmlNodesFor` option of `react-i18next`?
Allowing all HTML is not safe.

Reply:
@yomybaby 
There is no option..
But I found that I can customize registerTranslateConfig()'s interplate option and I think that is work well.
I will apply and request review again

Reply:
@yomybaby 
I've searched for that feature, but lit-translate doesn't support it.
Therefore, I solved it using the method we discussed in the meeting